slash/react/src/Accordion/Header.tsx

```diff
@@ -25,8 +39,57 @@ const Header = ({ children, className, classModifier, id }: HeaderProps) => {
 
   return (
     <summary className={componentClassName} id={id}>
-      <h3 className="af-accordion__item-title">{children}</h3>
-      <span className="glyphicon glyphicon-menu-down" />
+      {type === "light" ? (
+        <Title heading="h3">
+          <Svg src={arrow} className="af-accordion__item-header-icon" />
+          {children}
+          {actions
+            ? actions
+                .filter((action): action is NonNullable<typeof action> =>
+                  Boolean(action),
+                )
+                .map(({ key, label, leftIcon, rightIcon, onClick }) => (
+                  <Button
+                    key={key}
+                    onClick={onClick}
+                    leftIcon={leftIcon ? <Svg src={leftIcon} /> : undefined}
+                    rightIcon={rightIcon ? <Svg src={rightIcon} /> : undefined}
+                    variant="ghost"
+                  >
```

Comment:
Ce serait pas mal de laisser les devs pouvoir surcharger d'autres props que simplement les 5 misent initialement non ? (pareil pour l'autre bouton aussi du coup)
```suggestion
                .map(({ label, leftIcon, rightIcon }) => (
                  <Button
                    leftIcon={leftIcon ? <Svg src={leftIcon} /> : undefined}
                    rightIcon={rightIcon ? <Svg src={rightIcon} /> : undefined}
                    variant="ghost"
                    {...props}
                  >
```

Reply:
Perso je suis pas fan du tout, on a un DS à respecter qui permet certains cas et une certaines souplesses. Entièrrement laisser la main à l'utilisateur sort de ce cadre, si l'utilisateur souhaite faire ce qu'il souhaite il peut éviter le composant React et simplement utiliser le CSS.

Reply:
Je suis d'accord et au début je limitais aussi beaucoup mes composants du DS.
Jusqu'à qu'on se rende compte qu'il y ai besoin d'une petite surcharge spécifique à notre cas (ça peut-être pour de l'accessibilité ou pour d'autres choses, mais toujours en restant dans les règles DS).
Après on peut très bien laisser comme c'est ici et si on se rend compte du besoin on avisera en fonction du contexte (même si c'est vrai que pour ce cas spécifique j'ai du mal à voir quels besoins de surcharge il pourrait y avoir 🤔).

Reply:
Oui je serais plus à KISS et voir au besoin. On a bcp itéré avec @adrien-dos sur le besoin notamment pour ajouter les icones etc et je pense qu'on couvre pas mal là, je resolve si ca te va dans ce sens